packages/rest/src/rest.server.ts

```diff
@@ -592,11 +594,19 @@ export class RestServer extends Context implements Server, HttpServerLike {
 
     const port = await this.get<number | undefined>(RestBindings.PORT);
     const host = await this.get<string | undefined>(RestBindings.HOST);
+    const protocol = await this.get<HttpProtocol>(RestBindings.PROTOCOL);
```

Comment:
I think the type case should not be necessary because `RestBindings.PROTOCOL` already provides the information about the type of the bound value. Could you please check?

```ts
const port = await this.get(RestBindings.PORT);
// etc.
const httpsOptions = await this.get(RestBindings.HTTPS_OPTIONS);
```